cinema/models.py

```diff
@@ -35,12 +40,25 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def create_custom_path(instance: "Movie", filename: str) -> pathlib.Path:
```

Comment:
I would rename this function. I think this name is to general but what it does is something more specific.